src/Html.php

```diff
@@ -804,17 +761,21 @@ private static function booleanInput($type, $name, $checked = false, $options =
      *   except that the array keys represent the optgroup labels specified in $items.
      * - encodeSpaces: bool, whether to encode spaces in option prompt and option value with `&nbsp;` character.
      *   Defaults to false.
-     * - encode: bool, whether to encode option prompt and option value characters.
-     *   Defaults to `true`.
+     * - encode: bool, whether to encode option prompt and option value characters. Defaults to `true`. This option is
```

Comment:
"This option is available" should be removed.

Reply:
If available as an option.

Reply:
What do you mean?

Reply:
- encode: bool, whether to encode option prompt and option value characters. Defaults to `true`.

Reply:
Yes. This part is understandable but "This option is available" doesn't add anything to the meaning so should be removed.